app/editor/src/components/form/wysiwyg/Wysiwyg.tsx

```diff
@@ -76,16 +84,20 @@ export const Wysiwyg: React.FC<IWysiwygProps> = ({ fieldName, label, required, e
 
   const stripHtml = () => {
     // strip html from string
-    let doc = new DOMParser().parseFromString(values[fieldName] as string, 'text/html');
-    setFieldValue(fieldName, doc.body.textContent || '');
-    setState({ ...state, html: doc.body.textContent || '' });
+    if (!!fieldName) {
+      let doc = new DOMParser().parseFromString(values[fieldName] as string, 'text/html');
+      setFieldValue(fieldName, doc.body.textContent || '');
+      setState({ ...state, html: doc.body.textContent || '' });
```

Comment:
@jdtoombs Should the `setState(...)` occur even if `fieldName == undefined`?

Reply:
I think it should be fine in our case since our WYSIWYG will always be tied to formik